packages/react-popup/package.json

```
@@ -15,6 +15,7 @@
     "build": "just-scripts build",
     "clean": "just-scripts clean",
     "code-style": "just-scripts code-style",
+    "e2e": "echo 'enabled when added to suite package'",
```

```suggestion
    "e2e": "e2e",
```

Should be this modified?

---

nope, since e2e runs on storybooks, it can only be a thing when a component is added to the converged suite :(